<launch>
    <include file="$(find multi-robot-exploration-pkg)/launch/move_base.launch">
        <arg name="ns" value="tb3_0"/>
    </include>
    <include file="$(find multi-robot-exploration-pkg)/launch/move_base.launch">
        <arg name="ns" value="tb3_1"/>
    </include>
    <include file="$(find multi-robot-exploration-pkg)/launch/move_base.launch">
        <arg name="ns" value="tb3_2"/>
    </include>
    <include file="$(find multi-robot-exploration-pkg)/launch/move_base.launch">
        <arg name="ns" value="tb3_3"/>
    </include>
    <include file="$(find multi-robot-exploration-pkg)/launch/move_base.launch">
        <arg name="ns" value="tb3_4"/>
    </include>
</launch>